packages/server/lib/modes/interactive.ts

```diff
@@ -125,6 +125,7 @@ export = {
         show: true,
         frame: true,
         transparent: false,
+        // @ts-ignore
```

Comment:
cannot find getPathToIcon - I think there's something strange with the icons package export/import